AK/StringView.h

```diff
@@ -176,8 +177,14 @@ class StringView {
     {
         VERIFY(!separator.is_empty());
         // FIXME: This can't go in the template header since declval won't allow the incomplete StringView type.
```

Comment:
...? this FIXME looks very out of date. it *is* in the header.

Reply:
The command says it wants it to move into the `template<typename Callable, bool ReturnsError>` if I understand it correctly

Reply:
oh, the line below, not the entire function. got it. don't mind me.